src/libumf.def.in

```diff
@@ -33,6 +33,12 @@ EXPORTS
     umfMemoryProviderPurgeForce
     umfMemoryProviderPurgeLazy
     umfMemoryProviderPutIPCHandle
+    umfMempolicyBind;
+    umfMempolicyCreate;
+    umfMempolicyDestroy;
+    umfMempolicyInterleave;
+    umfMempolicyPreferred;
+    umfMempolicySetInterleavePartSize;
```

Comment:
maybe better name would be umfMempolicyInterleaveSetPartSize ? it also looks better when the names are sorted

Reply:
in your option we are creating new namespace (umf::Mempolicy::Interleave::set::Partsize) in future we will have common fields in mempolicy so then we will have umf::mempolicy::set::someCommonField. Not sure if this will be to chaotic for user.

Reply:
ok, so the parameter here is not just "PartSize" but the whole "InterleavePartSize" - got it